roles/openshift_storage_glusterfs/tasks/glusterfs_uninstall.yml

```diff
@@ -1,6 +1,6 @@
 ---
 
-- name: Delete pre-existing heketi resources
+- name: Delete pre-existing Heketi resources
```

Comment:
heketi is intentionally all lower-case. :)

Reply:
okay but then again [this](https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_storage_glusterfs/tasks/glusterfs_uninstall.yml#L67) should be lower-case too? :)

Reply:
Yup! Not all developers are consistent about it, but officially it is.

Reply:
@jarrpa in my env (3.7 branch) the dir was still present, with the change the dir was gone. And i'll run / put a simple test case and attach the output here so you can see it